content/applications/hr/employees/new_employee.rst

```diff
@@ -79,188 +62,271 @@ Resumé tab
+The Work Information tab is where the employee's specific job related information is housed. Their
+working schedule, various roles, who approves their specific requests (time off, timesheets, and
+expenses), and specific work location details are listed here. Enter the following information for
+the new employee.
+
+- :guilabel:`Location`: Select the :guilabel:`Work Address` and :guilabel:`Work Location` from the
+  corresponding drop-down menus. The work address :guilabel:`Internal Link` button opens up the
+  selected company form in a window, and allows for editing. The :guilabel:`Work Location` is the
+  specific location details, such as a floor or building. If a new work location is needed, add the
+  location by typing it in the field.
+- :guilabel:`Approvers`: Using the drop-down menus, select the users responsible for approving
+  :guilabel:`Expenses`, :guilabel:`Time Off`, and :guilabel:`Timesheets` for the employee. The
+  :guilabel:`Internal Link` button opens a form with the approver's :guilabel:`Name`,
   :guilabel:`Email Address`, :guilabel:`Company`, :guilabel:`Phone`, and :guilabel:`Mobile` fields.
-  These can be modified, if needed. Click :guilabel:`Save` after making any edits.
-- :guilabel:`Schedule`: Select the :guilabel:`Working Hours` (required) and :guilabel:`Timezone` for
-  the employee. The :guilabel:`External Link` button opens up a detailed view of the specific daily
-  working hours. Working hours can be created, modified, or deleted here. Global time off (such as
-  holidays) can be entered in the :guilabel:`Global Time Off` tab. Click :guilabel:`Add a line` to
-  add a new global time off.
-- :guilabel:`Planning`: Click on a planning role from the drop-down menu for both the
-  :guilabel:`Default Planning Role` and the :guilabel:`Planning Roles` fields to add a role. There
-  is no limit to the amount of :guilabel:`Planning Roles` that can be selected for an employee, but
-  there can only be one :guilabel:`Default Planning Role`. The default is the *typical* role that
-  the employee performs, where the :guilabel:`Planning Roles` are *all* the specific roles the
-  employee is able to perform.
-
-  .. image:: new_employee/work-info.png
-     :align: center
-     :alt: Add the work information to the Work Information tab.
-
-Private information tab
+  These can be modified, if needed.
+- :guilabel:`Schedule`: Select the :guilabel:`Working Hours` and :guilabel:`Timezone` (both
+  required) for the employee. The :guilabel:`Internal Link` button opens up a detailed view of the
+  specific daily working hours. Working hours can be modified or deleted here.
+- :guilabel:`Planning`: The :guilabel:`Planning` section affects the *Planning* app, and will only
+  appear if the *Planning* app is installed. Click on a role from the drop-down menu for both the
+  :guilabel:`Roles` and the :guilabel:`Default Role` fields to add a role. There is no limit to the
```

Comment:
In 16.3 they changed it to just default role, and I was using 16.4. I feel this is clear enough for people following along if they are in an earlier version, since only one field says 'default'.